tests/homeCounter.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import counter from '../src/modules/counter.js';
+import displayPokemon from '../__mocks__/likeAPiMock.js';
+
```

Comment:
- [ ] For better readability of your code, I suggest that you put all the tests in one folder instead of creating two folders for tests.